### ISSUE TITLE
AccessibilityInfo.setAccessibilityFocus is not implemented for TouchableWithoutFeedback

### ISSUE THREAD

**Issue by MicrosoftNik** [microsoft/react-native-windows#8498]
Have been trying to set the accessibility focus of a TouchableWithoutFeedback component upon rendering of the page.

```
const refT = React.useRef(null);

React.useEffect(() => {
  if (false && refT && refT.current) {
    const reactTag = findNodeHandle(refT.current) || 0;
    AccessibilityInfo.setAccessibilityFocus(reactTag);
  }
}, [refT]);

return (
  <TouchableWithoutFeedback
    ref={refT}
...
```

React Native throws the error:
`unable to get property typeViewFocused of undefined or null reference.`

Tried with react-native-windows versions 0.64.14 and 0.65.0.

**Comment by chrisglein** [comment]
> React Native throws the error:
unable to get property typeViewFocused of undefined or null reference.

Error may be on the JS side before it even gets to native code.

**Comment by chiaramooney** [comment]
From testing on an RNW app running 0.65, can confirm that the method will not work for some TouchableWithoutFeedback cases. This is because TouchableWithoutFeedback forwards all accessibility configs to the child element. Thus if the child element is Text (which is not focusable), then setAccessibilityFocus will be a no-op. Work around for this is wrapping the child Text component in a View. Views are focusable and so the method will then work as expected. 

Only curious thing is that I am not able to replicate the error message you mentioned in your issue description. That error message is expected for an app running on 0.64 since the method was not implemented yet, but does not occur for apps running on 0.65. I would confirm that your app was upgraded to 0.65 correctly and that you now have the updated RNW source. 

Closing issue for now.
